test/detailed_balance/CMakeLists.txt

```diff
@@ -240,6 +239,32 @@ one_balance_test("pi_rho_f2"
     "300.0"
 )
 
+########## multi-particle reactions ########
+
+one_balance_test("multi_pi3_omega"
+    "π⁰,π⁺,π⁻,ω"
+    "π⁰,π⁺,π⁻:ω"
+    "150.0"
+)
+
+one_balance_test("multi_pi3_phi"
+    "π⁰,π⁺,π⁻,φ"
+    "π⁰,π⁺,π⁻:φ"
+    "150.0"
+ )
+
+one_balance_test("multi_pi2_eta_etaprime"
+    "π⁰,π⁺,π⁻,η,η'"
+    "π⁺,π⁻,η:η'|π⁰,π⁰,η:η'"
+    "2500.0"
+ )
+
+ one_balance_test("multi_N_pi_deutron"
+    "π⁺,π⁻,π⁰,N⁺,N⁰,d,N̅⁰,N̅⁻,d̅"
+    "π⁺,d:N⁺,N⁺|π⁰,d:N⁺,N⁰|π⁻,d:N⁰,N⁰|π⁺,d̅:N̅⁰,N̅⁰|π⁻,d̅:N̅⁻,N̅⁻|π⁰,d̅:N̅⁻,N̅⁰|π⁺,d:π⁺,N⁺,N⁰|π⁰,d:π⁰,N⁺,N⁰|π⁻,d:π⁻,N⁺,N⁰|N⁺,d:N⁺,N⁺,N⁰|N⁰,d:N⁰,N⁺,N⁰|N⁰,d̅:N⁰,N̅⁻,N̅⁰|N⁺,d̅:N⁺,N̅⁻,N̅⁰|N̅⁻,d:N̅⁻,N⁺,N⁰|N̅⁰,d:N̅⁰,N⁺,N⁰|π⁰,d̅:π⁰,N̅⁻,N̅⁰|π⁺,d̅:π⁺,N̅⁻,N̅⁰|π⁻,d̅:π⁻,N̅⁻,N̅⁰|N̅⁰,d̅:N̅⁰,N̅⁻,N̅⁰|N̅⁻,d̅:N̅⁻,N̅⁻,N̅⁰"
```

Comment:
I trust you here, that these are all the accessible deuteron channels...

Reply:
ok :) they also check themselves, since otherwise the average over the iospin group would not be correct, I think.